ext/curl/interface.c

```diff
@@ -909,6 +909,18 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSL);
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSPI);
 
+#if LIBCURL_VERSION_NUM >= 0x071306 /* Available since 7.19.6 */
+	REGISTER_CURL_CONSTANT(CURL_VERSION_CURLDEBUG);
```

Comment:
As this is an old constant, I wonder if it was omitted intentionally. The curl docs do say "This is mainly of interest for libcurl hackers". But I don't see any particular problem with exposing it either...

Reply:
ping @bagder for thoughts about exposing it

Reply:
It's indeed part of the public API, exposed to the world just like the other bits there so I have no objections.

If it actually is *useful* to anyone is another matter...